text/0025-purchase-order.md

```diff
@@ -551,8 +552,9 @@ message PurchaseOrderPayload {
 }
 
 message CreatePurchaseOrderPayload {
-  string uuid = 1;
-  uint64 created_at = 2;
+  string org_id = 1;
+  string uuid = 2;
+  uint64 created_at = 3;
 }
 
 message UpdatePurchaseOrderPayload {
```

Comment:
Would this message also need a `po_uuid`?

Reply:
That is just the `uuid` here

Reply:
I'm slightly confused where the `uuid` is coming from in this message?